docs/CI.md

```diff
@@ -30,24 +30,39 @@ More specifically, the following workflows are included:
 
-## Github Actions for firebase hosting
+### Download your bundled executable to install and run your task
 
-Automatic deployment on a firebase hosting site can also be setup with github actions. Before proceeding with the follwing steps, please make sure that firebase is configured by following the [Getting started with firebase section under Set Up Firebase](firebase.md)
+You can download the executable file from either the tagged release page or the GitHub actions page. To install, unzip the downloaded file, install and run.
 
-- Run `firebase init hosting:github` in the terminal and follow the CLI prompts to start setting up the github actions.
-  - For the github directory question, select your task github directory.
-  - When prompted with the npm command, type `npm install && npm run build:firebase`
+The executable does not require installation of any additional software.
 
-There should be two new `.yml` files, `firebase-hosting-pull-request.yml` and `firebase-hosting-merge.yml`, created in the `.github/workflows` directory for a deployment preview for each pull request and the official deployment when merged onto the main branch.
+### Uninstall the task
 
-## Download your bundled executable to install and run your task
+#### Windows
 
-You can download the executable file from either the tagged release page or the GitHub actions page. To install, unzip the downloaded file, install and run.
+Go to System Settings: Add or remove programs. Look for the name of your task in the installed programs list, and Uninstall.
 
 The executable does not require installation of any additional software.
 
-## Uninstall the task
+## Firebase Hosting
 
-### Windows
+Automatic deployment on a firebase hosting site can also be setup with github actions. Before proceeding with the following steps, please make sure that firebase is configured by following the [Getting started with firebase section under Set Up Firebase](firebase.md). Then run the following command in the terminal:
 
-Go to System Settings: Add or remove programs. Look for the name of your task in the installed programs list, and Uninstall.
+```console
+firebase init hosting:github
+```
+
+- Be sure to type "`<username>/<repository name>`" exactly as it appears in GitHub when the prompt "For which GitHub repository would you like to set up a GitHub workflow?" appears.
+- Enter "`y`" for the prompt "Set up the workflow to run a build script before every deploy?"
+- Enter the following command for the prompt "What script should be run before every deploy?":
```

Comment:
I think you can reverse these sentences. Putting the question first then the answer